src/screens/DiscoveryScreenV2/index.js

```diff
@@ -302,7 +302,7 @@ const DiscoveryScreenV2 = ({route}) => {
 
   return (
     <DiscoveryContainer>
-      <StatusBar translucent={false} />
+      <StatusBar translucent={false} barStyle={'light-content'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `StatusBar` component's `barStyle` prop from default to `'light-content'` is a good move if you want to ensure that the status bar content is visible against a dark background. However, please make sure that this change doesn't affect readability on screens where the background might be lighter.

```diff
- <StatusBar translucent={false} />
+ <StatusBar translucent={false} barStyle={'light-content'} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->